optimum/onnxruntime/quantization.py

```diff
@@ -279,7 +279,7 @@ def export(
         quantization_config: QuantizationConfig,
         calibration_tensors_range: Optional[Dict[NodeName, Tuple[float, float]]] = None,
         use_external_data_format: bool = False,
-        quantization_preprocessor: Optional[QuantizationPreprocessor] = None,
+        preprocessor: Optional[QuantizationPreprocessor] = None,
```

Comment:
wouldn't this break backward compatibility?

Reply:
Yep it would, that's why I pushed a second commit to revert this. I think you did the review before I pushed this second commit, now this change should not appear anymore.